src/resources/styles.ts

```diff
@@ -59,6 +60,11 @@ export const haStyle = css`
     background-color: var(--primary-background-color);
   }
 
+  app-toolbar {
+    border-bottom: 1px solid var(--divider-color, transparent);
```

Comment:
Not sure why we need this border on the toolbar, I don't think it will look good.

Reply:
What does this do for the config pages?
![image](https://user-images.githubusercontent.com/5662298/78063963-4d25da80-7391-11ea-9b49-929a55964e76.png)

Reply:
And I don't think this looks better
![image](https://user-images.githubusercontent.com/5662298/78064476-1c927080-7392-11ea-87da-e9dd74367108.png)

Reply:
Your first image is because of .toolbar is again 65px... i didn't checked this (second commit for home assistant) I can fix this also but....
It sounds like if you dont like it, it wont be merged? ;)

This is how it is without my modifications lol
![VirtualBox_HOMEASSISTANT_31_03_2020_21_07_50](https://user-images.githubusercontent.com/11946839/78065608-d3432080-7393-11ea-83de-943774465674.png)

Reply:
Well I'm not doing this alone, but yeah I do have a voice in it :-)

Reply:
fair enough